### PR TITLE
8.3 release (website + release notes)

### PR DESCRIPTION
**Pull request by stormi**
Release notes for XCP-ng 8.3. Already checked for language correctness in theory. Links all verified. 

Next I'll also have to update various parts of the docs that mention XCP-ng 8.2 or 8.3.

> Before submitting the pull request, you must agree with the following statements by checking both boxes with a 'x'.
> * [x] "I accept that my contribution is placed under the CC BY-SA 2.0 license [1]."
> * [x] "My contribution complies with the Developer Certificate of Origin [2]." 
>
> [1] https://creativecommons.org/licenses/by-sa/2.0/
> [2] https://docs.xcp-ng.org/project/contributing/#developer-certificate-of-origin-dco

### CODE REVIEW

File: docs/releases/release-8-3.md [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+- Users who choose to modify their SSH configuration will need to maintain these settings themselves, as any updates provided by our Security Team through new builds will be overridden by user-defined settings.
+
+### Restricted use of port 80 [XS]
+
+To enhance security, it is now possible to close TCP port 80 on the management interface. By default, port 80 remains open for specific clients that may still require it, but all internal connections now utilize HTTPS. Previously, VM migration occurred over port 80, but this has been updated.
+
+To close port 80, run the following command:
+```
+xe pool-param-set uuid=<pool-uuid> https-only=true
+```
```
Comment (bleader) [inline docs/releases/release-8-3.md:213]: Should we add a mention that this won't do it for IPv6 for now? Or do we expect the fix to be ready by release time?

File: docs/releases/release-8-3.md [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+
+### VM anti-affinity [XS]
+
+XenServer added a new feature which allows to constitute anti-affinity VM groups, meaning that you would like to avoid VMs to be started, or evacuated from another host, towards a host where another VM of the same group already runs.
+
+For more information, refer to [XenServer's documentation](https://docs.xenserver.com/en-us/xenserver/8/vms/placement).
+
+### PCI passthrough support in the API
+
+We expanded the XAPI API by adding endpoints to manage PCI passthrough devices. Xen Orchestra developers added corresponding UI elements, enabling users to add or remove devices from the passthrough list. Once a reboot of the host, these devices are released by dom0 (the control domain) and become available for assignment to VMs.
```
Comment (bleader) [inline docs/releases/release-8-3.md:246]: `Once a reboot of the host` +has been done, maybe ? Or once the host has rebooted.

File: docs/releases/release-8-3.md [hunk 3]
```diff
@@ -0,0 +1,417 @@
...
+- Almost all Python components have been ported to Python 3.
+- VLAN display support has been added to `xsconsole` (contributed upstream to the XAPI project).
+- `smartmontools` was updated to version 7, adding JSON export. We also added a XAPI plugin used by Xen Orchestra to display disk status.
+- Updated guest templates, including two new generic Linux templates (BIOS and UEFI).
+- The web server provided by XAPI now reports the correct MIME type for SVG (and other) files, a requirement for XO Lite.
+- Migration stream compression (XS).
+- IPv6 support for VM network booting (XS).
+- Numerous under-the-hood changes and fixes.
+
+We've also invested heavily in test automation to improve coverage. Nevertheless, feedback from our user community remains invaluable ((and really was during all the development of XCP-ng 8.3!).
```
Comment (bleader) [inline docs/releases/release-8-3.md:281]: double (

File: docs/releases/release-8-3.md [hunk 4]
```diff
@@ -0,0 +1,417 @@
...
+### Legacy partitioning layout
+
+XCP-ng 8.3 doesn't support the old partition layout where the system partition only took 4 GB on disk, and GPT partitioning is now mandatory. If you need to upgrade a pool whose hosts still use this partitioning scheme, what to do depends on the exact situation.
+
+* If the partitioning type is GPT, see [using the new partitions](../../installation/upgrade/#using-the-new-partitions).
+* If the partitioning type is MBR, a fresh installation is necessary. The presence of a Dell Utility Partition also prevents upgrading (and is usually coupled with an MBR partitioning type).
+
+### Nested virtualization (though not on purpose)
+
+Upgrading Xen from version 13 to version 17 had a lot of benefits, but it came with one drawback: nested virtualization, which had always been experimental but was at least working in some useful situations (such as testing XCP-ng inside XCP-ng), is now non-functional due to fundamental changes in the codebase.
```
Comment (bleader) [inline docs/releases/release-8-3.md:322]: why not use 4.13 and 4.17 instead of version 13 and version 17 ?

Comment (stormi) [reply docs/releases/release-8-3.md:322]: I don't know :sweat:

File: docs/releases/release-8-3.md (hunk 1 above)
Comment (stormi) [reply docs/releases/release-8-3.md:213]: Let's mention it in the known issues section.

File: docs/releases/release-8-3.md (hunk 2 above)
Comment (stormi) [reply docs/releases/release-8-3.md:246]: Fixed

File: docs/releases/release-8-3.md (hunk 3 above)
Comment (stormi) [reply docs/releases/release-8-3.md:281]: Fixed

File: docs/releases/release-8-3.md (hunk 4 above)
Comment (stormi) [reply docs/releases/release-8-3.md:322]: Fixed

File: docs/releases/release-8-3.md [hunk 5]
```diff
@@ -0,0 +1,417 @@
+---
+sidebar_position: 1
+---
+
+# XCP-ng 8.3
+
+We are proud to present XCP-ng 8.3, the last release of the 8.x platform.
```
Comment (ydirson) [inline docs/releases/release-8-3.md:7]: "which will be the last release" ?

File: docs/releases/release-8-3.md [hunk 6]
```diff
@@ -0,0 +1,417 @@
...
+```
+
+### Improvements in the installer
+
+Two years ago, development around XenServer's (and XCP-ng's) installer [was opened](https://github.com/xenserver/host-installer), allowing us to introduce several improvements and fixes (while previously we could modify the installer, contributing upstream reduces the long-term maintenance burden of divergence between our version and theirs).
+
+However, despite good intentions on both sides, not every change we submitted was accepted, leading to some remaining divergences. Unfortunately, some improvements we developed couldn’t be included in XCP-ng 8.3 as they would have caused significant differences with the upstream base (keeping large patches up to date as the base code evolves is not sustainable).
+
+Key improvements made to the installer include:
+- `memtest86+` has been updated to version 6 with EFI support.
```
Comment (ydirson) [inline docs/releases/release-8-3.md:222]: While the first part mentions only host-installer, which was the opened one, memtest is related to the install ISO, whose production process was not.  Also mention `create-install-image`?

**Review by gduperrey — COMMENTED** [review]
lgtm

File: docs/releases/release-8-3.md (hunk 6 above)
Comment (stormi) [reply docs/releases/release-8-3.md:222]: Agreed. Can you propose a rewording?

File: docs/releases/release-8-3.md (hunk 5 above)
Comment (Danp2) [reply docs/releases/release-8-3.md:7]: I think this should be `latest` instead.

Comment (stormi) [reply docs/releases/release-8-3.md:7]: Done

Comment (stormi) [reply docs/releases/release-8-3.md]: Oops, wrong comment. Actually I really mean `last`, and Yann proposed to improve the sentence to "which will be the last release of the 8.x platform".

File: docs/releases/release-8-3.md (hunk 6 above)
Comment (ydirson) [reply docs/releases/release-8-3.md:222]: proposal:
* "..., not every change we submitted was accepted, and the XS image-creation process is still not open, ..."
* new paragraph:

> We redesigned the image-creation process to [use a brand new set of scripts](https://github.com/xcp-ng/create-install-image/) - here again, this effort which started for 8.3 was already put to good use last year with XCP-ng 8.2.1's latest refreshed installation ISO.  Users willing to customize the installation images also have a more light-weight option, with our [iso-remaster script](https://github.com/xcp-ng/xcp/tree/master/scripts/iso-remaster).  And those facing a missing hardware driver at installation time will also have the option of [producing a Driver Disk](https://github.com/xcp-ng/driver-disks) to complement a pristine installation image.

Comment (stormi) [reply docs/releases/release-8-3.md]: Added, thanks!

**Comment by stormi** [comment]
I leave it as a draft until I have squashed the commits (leaving them separate for now in case some want to review the additions).

File: docs/releases/release-8-3.md [hunk 7]
```diff
@@ -0,0 +1,483 @@
...
+
+* If the partitioning type is GPT, see [using the new partitions](../../installation/upgrade/#using-the-new-partitions).
+* If the partitioning type is MBR, a fresh installation is necessary. The presence of a Dell Utility Partition also prevents upgrading (and is usually coupled with an MBR partitioning type).
+
+### Nested virtualization (though not on purpose)
+
+Upgrading Xen from version 4.13 to version 4.17 had a lot of benefits, but it came with one drawback: nested virtualization, which had always been experimental but was at least working in some useful situations (such as testing XCP-ng inside XCP-ng), is now non-functional due to fundamental changes in the codebase.
+
+What needs to be done in the codebase for complete nested virtualization support was explained to the Xen developer community during XenSummit by Georges Dunlap, but it's not a small task, so we don't know at the moment when nested virtualization will be fully implemented.
```
Comment (olivierlambert) [inline docs/releases/release-8-3.md:378]: Link to the YT video here

File: docs/releases/release-8-3.md [hunk 8]
```diff
@@ -0,0 +1,483 @@
...
+
+Though not all issues from XenServer 8 apply to XCP-ng 8.3, some of the information and tips provided in [XenServer 8's Known Issues](https://docs.xenserver.com/en-us/xenserver/8/whats-new/known-issues) might still be useful for certain scenarios.
+
+### Cross-pool live migration from XenServer < 7.1
+
+Live migrating a VM from an older XenServer version can sometimes result in errors, leading to the following consequences:
+* The VM may reboot.
+* The VM may get duplicated, meaning the same VM UUID (and usually its VDIs too) will appear both on the sender and receiver host. In such a case, you'll need to remove the duplicate from the receiver host.
+
+It would require a hotfix to the old XenServer, but since those versions are no longer supported, XenServer won't be developing one.
```
Comment (olivierlambert) [inline docs/releases/release-8-3.md:483]: Tell that Xen Orchestra warm migration is the solution (you can link https://xen-orchestra.com/blog/warm-migration-with-xen-orchestra/)

**Pull request merged**